### DISCUSSION TITLE
TTS not working on arch Linux

### DISCUSSION THREAD

**Discussion by Tribal64** [karashiiro/TextToTalk#178] [Q&A]
Hey how are yous TTS plugin has trouble installing and deleting I would just like it to work so I can play thanks.

**Comment by cimba007** [comment]
Go the same problem. Plugin will not install on FF14-Launcher version on steamdeck (Discover Appstore Version of FF14-Launcher).

**Comment by karashiiro** [comment]
There's an open issue for this: https://github.com/karashiiro/TextToTalk/issues/151
